zuul.d/arista-eos-jobs.yaml

```diff
@@ -69,213 +49,102 @@
-      ansible_test_python: 2.7
-
-- job:
-    name: ansible-test-network-integration-eos-network_cli-python27
-    parent: ansible-test-network-integration-eos-network_cli
-    nodeset: eos-4.24.6-python27
-    vars:
-      ansible_test_python: 2.7
+# =============================================================================
 
 - job:
-    name: ansible-test-network-integration-eos-httpapi-python27-stable211
-    parent: ansible-test-network-integration-eos-httpapi
-    nodeset: eos-4.24.6-python27
-    required-projects:
-      - name: github.com/ansible/ansible
-        override-checkout: stable-2.11
-    vars:
-      ansible_test_python: 2.7
-
-- job:
-    name: ansible-test-network-integration-eos-network_cli-python27-stable211
-    parent: ansible-test-network-integration-eos-network_cli
-    nodeset: eos-4.24.6-python27
-    required-projects:
-      - name: github.com/ansible/ansible
-        override-checkout: stable-2.11
-    vars:
-      ansible_test_python: 2.7
-      ansible_test_split_in: 2
-
-- job:
-    name: ansible-test-network-integration-eos-network_cli-python27-stable211-scenario01
-    parent: ansible-test-network-integration-eos-network_cli-python27-stable211
-    vars:
-      ansible_test_do_number: 1
-
-- job:
-    name: ansible-test-network-integration-eos-network_cli-python27-stable211-scenario02
-    parent: ansible-test-network-integration-eos-network_cli-python27-stable211
-    vars:
-      ansible_test_do_number: 2
-
-- job:
-    name: ansible-test-network-integration-eos-httpapi-python27-stable29
-    parent: ansible-test-network-integration-eos-httpapi
-    nodeset: eos-4.24.6-python27
-    required-projects:
-      - name: github.com/ansible/ansible
-        override-checkout: stable-2.9
-    vars:
-      ansible_test_python: 2.7
-
-- job:
-    name: ansible-test-network-integration-eos-network_cli-python27-stable29
-    parent: ansible-test-network-integration-eos-network_cli
-    nodeset: eos-4.24.6-python27
-    required-projects:
-      - name: github.com/ansible/ansible
-        override-checkout: stable-2.9
-    vars:
-      ansible_test_python: 2.7
-
-- job:
-    name: ansible-test-network-integration-eos-python35
-    parent: ansible-test-network-integration-eos
-    nodeset: eos-4.24.6-python35
-    vars:
-      ansible_test_python: 3.5
+    name: network-ee-integration-tests-eos
+    parent: ansible-network-eos-appliance
 
 - job:
-    name: ansible-test-network-integration-eos-python36
-    parent: ansible-test-network-integration-eos
-    nodeset: eos-4.24.6-python36
+    name: network-ee-integration-tests-eos
+    parent: network-ee-integration-tests
+    nodeset: eos-4.24.6-python38
     vars:
-      ansible_test_python: 3.6
+      ansible_test_integration_targets: "eos_.*"
 
 - job:
-    name: ansible-test-network-integration-eos-httpapi-python36
-    parent: ansible-test-network-integration-eos-httpapi
-    nodeset: eos-4.24.6-python36
+    name: network-ee-integration-tests-eos-network_cli
```

Comment:
given we don't callout network_cli for other collection jobs, I think we could drop it here and assume it is the default connection to test with, for network